scripts/scil_filter_tractogram_anatomically.py

```diff
@@ -118,17 +207,27 @@ def main():
     o_dict['streamline_count_after_filtering_length'] =\
         len(length_sft.streamlines)
 
+    if args.save_outliers:
+        outlier = compute_outliers(sft, length_sft)
+        o_dict['streamline_count_discarded_when_filtering_length'] =\
+            len(outlier.streamlines)
+        save_outliers(out_path, 'length', 'length', in_tracto_name, ext,
+            outlier, length_sft)
+
+    if len(length_sft.streamlines) == 0:
+        if args.no_empty:
+            logging.debug("The file {} won't be written (0 streamline after "
```

Comment:
streamline : streamlines.

Same comment for all appearances of "0 streamline".

Reply:
Done in dbd0b2a.